cypress/support/interceptors.js

```diff
@@ -8,4 +9,13 @@ export const activationKeysInterceptors = {
             body: fixtures
         }).as('getActivationKeys');
     }
+}
+
+export const singleActivationKeyInterceptor = {
+    keys: (fixtures = singleActivationKeyFixtures) => {
+        cy.intercept('GET', '', {
+            statusCode: 200,
+            body: fixtures
+        }).as('getSingleActivationKey');
+    } 
```

Comment:
I would suggest slightly different and more descriptive wording

```suggestion
export const activationKeysInterceptors = {
   'successful with one key': (fixtures = singleActivationKeyFixtures) => {
        cy.intercept('GET', '', {
            statusCode: 200,
            body: fixtures
        }).as('getSingleActivationKey');
    } 
```